arcade/camera.py

```diff
@@ -184,16 +186,21 @@ def get_map_coordinates(self, camera_vector: Union[Vec2, tuple]) -> Vec2:
         """
         return Vec2(*self.position) + Vec2(*camera_vector)
 
-    def resize(self, viewport_width: int, viewport_height: int) -> None:
+    def resize(self, viewport_width: int, viewport_height: int, *,
+               resize_projection: bool = False) -> None:
```

Comment:
Can we set this to True instead to have better backwards compatibility?

Reply:
Done